docs/dashboards/synchronize-components.md

```diff
@@ -202,5 +199,5 @@ class CustomComponent extends Component {
 }
 ```
 
-The demo below shows a custom sync connecting the Highcharts Component with a custom component:
+The below example shows how `custom sync` between a `Highcharts Component` and a `custom component` works:
```

Comment:
Instead of \` it is better to put URLs here. Instead of enclosing words in \` it is better to put URLs here. Instead of enclosing words in \` it is better to put URLs here. Most of them are not literal keywords from the code that should be quoted with \`.